recipes/cppy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "cppy" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4eda6f1952054a270f32dc11df7c5e24b259a09fddf7bfaa5f33df9fb4a29642
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - cppy
```

Comment:
Not sure if we should add a test to make sure the header files are included properly. Something like:

```
test:
    imports:
        - cppy
    commands:
        - test -f ${SP_DIR}/cppy/include/cppy/cppy.h
```

This might not work on the windows build, but if this is `noarch: python` it will only get built on linux once it hits the feedstock, I believe.